examples/panda/panda_goal_reaching/controllers/robot_supervisor_manager/DDPG_runner.py

```diff
@@ -0,0 +1,85 @@
+from numpy import convolve, ones, mean, random
+
+from robot_supervisor_ddpg import PandaRobotSupervisor
+from agent.ddpg import DDPGAgent
+
+
+def run():
+    # Initialize supervisor object
+    env = PandaRobotSupervisor()
+
+    # The agent used here is trained with the DDPG algorithm (https://arxiv.org/abs/1509.02971).
+    # We pass (10, ) as numberOfInputs and (7, ) as numberOfOutputs, taken from the gym spaces
+    agent = DDPGAgent(alpha=0.000025, beta=0.00025, input_dims=[env.observation_space.shape[0]], tau=0.001, batch_size=64,  layer1_size=400, layer2_size=400, n_actions=env.action_space.shape[0]) 
+              
+    # agent.load_models() # Load the pretrained model
+    episodeCount = 0 
+    episodeLimit = 50000
```

Comment:
This might be a universal constant value or argument of run function?

Reply:
<code>episodeCount = 0</code> and <code>episodeLimit = 50000</code> are set as universal constants.
But we still can change these constants when doing other tasks.
Similar to these [lines](https://github.com/aidudezzz/deepworlds/blob/5f86cdd69b4e9906b43ccc4acc3d31b3e60125b9/examples/cartpole/cartpole_discrete/controllers/robot_supervisor_manager/PPO_runner.py#L16) in cartpole_discrete example.

Reply:
I meant that this can be on top of the file (bellow imports something like `EPISODE_LIMIT`) in order to be visible to the end user. Indeed this is as in cartpole, maybe we should change there as well. Of course, it is really minor, if you feel that's better as it is, feel free to resolve the comment